packages/html/src/templates/html.css

```diff
@@ -0,0 +1,179 @@
+:root {
```

Comment:
Might be worth checking out of we would like to change some settings when printing a html document using a media query:
```css
@media print {
  ...
}
But, I don't have any particular changes in mind!
```